utility_network/trace-a-utility-network/src/main/java/com/esri/samples/trace_a_utility_network/TraceAUtilityNetworkController.java

```diff
@@ -109,12 +109,12 @@ public void initialize() {
       mapView.setMap(map);
       // set the viewpoint to a subsection of the utility network
       mapView.setViewpointAsync(new Viewpoint(
-          new Envelope(-9813547.35557238, 5129980.36635111, -9813185.0602376, 5130215.41254146,
-              SpatialReferences.getWebMercator())));
+              new Envelope(-9813547.35557238, 5129980.36635111, -9813185.0602376, 5130215.41254146,
```

Comment:
these are just whitespace changes, but we shouldn't edit any other sample's files with this pr..
Just do a
`git checkout origin/master utility_network/trace-a-utility-network/src/main/java/com/esri/samples/trace_a_utility_network/TraceAUtilityNetworkController.java` and that should restore it 👍